app/views/users/inactive.haml

```diff
@@ -0,0 +1,30 @@
+%h1.title Inactive Users
+%table.table
+  %thead
+    %tr
+      %th Name
+      %th SPIRE
+      %th Email
+      %th Phone
+      %th
+      %th
+  %tbody
+    - @inactive.each do |user|
+      %tr
+        %td
+          = user.full_name
+          - if user == @fallback
+            (fallback)
```

Comment:
Why do we need the information about fallbacks if they're inactive?

Reply:
Quite. Indeed we should probably enforce that inactive users cannot be the fallback.